.github/workflows/test.yml

```diff
@@ -42,6 +42,7 @@ jobs:
           tree;
           zip -0 ccov.zip `find . \( -name "$PROJECT_NAME_UNDERSCORE*.gc*" \) -print`;
           grcov ccov.zip -s . -t lcov --llvm --ignore-not-existing --ignore "/*" --ignore "tests/*" -o lcov.info;
+          cargo test $CARGO_OPTIONS --no-default-features
```

Comment:
We don't use ensure_no_std in the CI? We should build it too

Reply:
we are in the check lint file